ignite/pkg/cosmosgen/templates/vuex/store/vuex-root

```diff
@@ -1 +0,0 @@
-THIS FILE IS GENERATED AUTOMATICALLY. DO NOT DELETE.
```

Comment:
May I learn why vuex-root isn't needed anymore?

Reply:
I believe the path-finding works differently nowadays? doesn't seem to be a reason for it

Reply:
The old javascript generator used it to [locate](https://github.com/ignite/cli/blob/de41f0668263f7dd1ddd84dbfc5b602750adcf70/ignite/pkg/cosmosgen/generate_javascript.go#L206) where to define the vuex modules loader within the vuex store location. That requirement disappeared now that it was replace by a completely separate `ts-client` folder location.